docs/index.md

```diff
@@ -0,0 +1,41 @@
+# Arnold
+
+Arnold is a tool to deploy dockerized applications to
+[OpenShift](https://www.openshift.com/) with
+[Ansible](https://www.ansible.com/). It was built by France Université Numérique
+to ease its infrastructure deployment.
+
+The current work mainly focuses on the [Open edX MOOC
+platform](https://open.edx.org/), but it can be considered as a generic tool to
+deploy your dockerized applications.
+
+## Overview
+
+Arnold has been designed as a suite of Ansible playbooks and OpenShift object
+definition templates (Jinja2). We take advantage of the `openshift_raw` Ansible
+module to make Ansible talk with OpenShift.
+
+As a DevOps using this project, you will need to adapt OpenShift object
+templates to suite your needs or constraints and run playbooks to push your
+changes to your OpenShift instance that orchestrate your OpenEdx services.
+
+## Documentation
+
+### I. Installation
```

Comment:
sort the doc files in a folder structure that follows this index?

Reply:
Done